### PR TITLE
Exercise 4: Synonyms and Rainbows

### PR DESCRIPTION
**Pull request by Cathe313**
Loving the colors module!

### CODE REVIEW

File: exercise-4.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+              }
+              
+              if (data && data.adjective && data.adjective.syn) {
+                synAdj = pretty.prettyOutput(data.adjective.syn);
+              } 
+              else {
+                synAdj = '';
+              }
+        }
+    ).then(
```
Comment (ziad-saab) [inline exercise-4.js]: here it seems you are arbitrarily calling another `.then` when you could have done all this in the previous handler. Normally as long as you're not returning a promise, or ending a chain by returning a value, you can keep going in the same handler.

**Comment by ziad-saab** [comment]
@Cathe313 I'd like you to revise this and remove the unnecessary `.then` if only to make sure you don't have any confusion regarding its usage.

**Comment by Cathe313** [comment]
Ok, I revised it.

**Comment by ziad-saab** [comment]
:+1:

**Pull request closed without merging**